### ISSUE TITLE
Ensure Icon Button Has Accessible Labels

### ISSUE THREAD

**Issue by hawkticehurst** [microsoft/vscode-webview-ui-toolkit#175]
### Describe the bug

Based on Storybook accessibility audits, must ensure that icon buttons have an accessible label.

### To reproduce

Look at the accessibility tab for the icon button story:

[https://microsoft.github.io/vscode-webview-ui-toolkit/?path=/story/library-button--with-icon-only](https://microsoft.github.io/vscode-webview-ui-toolkit/?path=/story/library-button--with-icon-only)

### Expected behavior

Icon buttons should have accessible labels.

### Screenshots

<img width="888" alt="Screen Shot 2021-08-16 at 11 40 38 AM" src="https://user-images.githubusercontent.com/39639992/129614231-8c771a01-a92b-46ca-83b7-bb08c1259588.png">

### Desktop (please complete the following information):

-   OS Version: `macOS 11.5`
-   Toolkit Version: `v0.6.2`
